app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] Since we are using a column that does not match the model (`author_id` instead of `user_id`), we need to specify the column name of the foreign key.

  ```
  has_many :comments, foreign_key: 'author_id'
  has_many :likes, foreign_key: 'author_id'
  ```